ql/src/test/org/apache/hadoop/hive/ql/TestTxnAddPartition.java

```diff
@@ -63,15 +63,15 @@ protected String getTestDataDir() {
 
 
   @Test
-  public void addPartition() throws Exception {
-
-    addPartition(false);
+  public void testAddPartition() throws Exception {
+    hiveConf.setBoolVar(HiveConf.ConfVars.HIVE_VECTORIZATION_ENABLED, false);
```

Comment:
why not do the config set in `addPartition` itself?

Reply:
The default value of `HiveConf.ConfVars.HIVE_VECTORIZATION_ENABLED` is `true` and this test doesn't change this at setup.

Reply:
i see you added the same for `testAddPartitionMM`. 👍